tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/legalize_to_linalg.cc

```diff
@@ -1569,8 +1571,11 @@ class ReduceConverter : public OpConversionPattern<lmhlo::ReduceOp> {
         dst_exprs.push_back(mlir::getAffineDimExpr(i, rewriter.getContext()));
       }
     }
-
-    auto maps = AffineMap::inferFromExprList({src_exprs, dst_exprs});
+    size_t operand_num = reduce_op.getOperation()->getNumOperands() / 3;
```

Comment:
nit: this should be `num_operands`?

I think it's clearer if you get the number from `adaptor`. Maybe `adaptor.inputs().size()`?